topics/elevate-invitations.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: site
+title: Elevate Invitations
+body_class: elevate-invitations
+---
+
+<dl>
+  <dt>Entity</dt>
+  <dd>Elevate Invitations</dd>
+
+  <dt>Description</dt>
+  <dd>Elevate invitations will contain invitations created either by Talis or by academics inviting other academics.</dd>
+
+  <dt>Database location</dt>
+  <dd><code>public.f_elevate_invitations</code></dd>
+</dl>
+
+
+### Columns
+
+| Column Name | Datatype | Description  | Example
+| --- | --- | --- | ---------- | 
+| `invitation_id` | `VARCHAR(64)` | The Talis ID of the invitation. Can be used to link this view with events. | `6200eae0790c89002bac4bd3` |
```

Comment:
Which events would be associated with invitations? Are these already in Census?

Reply:
Good catch, there are no events in Census associated with invitations so I'll remove that from the text.